lib/datasource/local/models/eos_action.dart

```diff
@@ -1,21 +1,34 @@
-import 'package:equatable/equatable.dart';
 import 'package:seeds/crypto/dart_esr/dart_esr.dart' as esr;
+import 'package:seeds/crypto/eosdart/eosdart.dart' as eos;
 
 /// Simple EOS Action data container
-class EOSAction extends Equatable {
-  final String accountName;
-  final String actionName;
-  final Map<String, dynamic> data;
+class EOSAction {
+  String? account;
+  String? name;
+  List<eos.Authorization?>? authorization;
+  Map<String, dynamic>? data;
 
-  bool get isValid => accountName.isNotEmpty && actionName.isNotEmpty;
+  bool get isValid => account != null && name != null && account!.isNotEmpty && name!.isNotEmpty;
 
-  const EOSAction({required this.accountName, required this.actionName, required this.data});
+  EOSAction() : super();
 
```

Comment:
super() ?

Reply:
That was leftover code. Removing and merging. Thanks!